python/ray/util/sgd/v2/trainer.py

```diff
@@ -170,6 +177,15 @@ def execute(self, func: Callable[..., T], *args, **kwargs) -> List[T]:
         """
         raise NotImplementedError
 
+    def start_training(
```

Comment:
Do we want to expose these as part of the user-facing `Trainer` API? Would we be able to directly call the executor instead?

Reply:
Good point. I created a new `_TuneTrainer` subclass (open to naming suggestions) to put these methods in.

Reply:
I think eventually this can be replaced with `trainer.run_generator`.